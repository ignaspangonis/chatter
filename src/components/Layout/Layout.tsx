import { ReactNode } from 'react'

type Props = {
  children: ReactNode
}

export default function Layout({ children }: Props) {
  return (
    <div className="w-full min-h-screen">
      <div className="flex w-full items-center justify-center py-large">
        <h1 className="text-4xl font-bold">Chatter</h1>
      </div>

      <hr className="border-cg5" />
      <div className="flex items-center justify-center max-w-x8-large mx-auto px-medium mt-x2-large">
        {children}
      </div>
    </div>
  )
}
