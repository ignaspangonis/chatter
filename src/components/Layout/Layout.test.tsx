import { render } from '@testing-library/react'

import Layout from './Layout'

describe('<Layout />', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Layout>
        <p>Content</p>
      </Layout>,
    )

    expect(container).toMatchSnapshot()
  })
})
