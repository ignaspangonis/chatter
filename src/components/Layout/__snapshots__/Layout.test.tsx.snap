// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> renders correctly 1`] = `
<div>
  <div
    class="w-full min-h-screen"
  >
    <div
      class="flex w-full items-center justify-center py-large"
    >
      <h1
        class="text-4xl font-bold"
      >
        Chatter
      </h1>
    </div>
    <hr
      class="border-cg5"
    />
    <div
      class="flex items-center justify-center max-w-[700px] mx-auto px-medium mt-x2-large"
    >
      <p>
        Content
      </p>
    </div>
  </div>
</div>
`;
